src/components/ColumnTitle/ColumnTitle.tsx

```diff
@@ -24,12 +25,22 @@ export const ColumnTitle: FunctionComponent<ColumnTitle> = (props) => {
         return <div>{title}</div>
     }
 
-    const filterable = props.rowMeta.filterable
-    const sort = <ColumnSort
+    const noSortable = props.widgetMeta.type === FieldType.multivalue
+        || props.widgetMeta.type === FieldType.multifield
+        || props.widgetMeta.type === FieldType.hidden
+        || props.widgetMeta.type === FieldType.fileUpload
+        || props.widgetMeta.type === FieldType.checkbox
```

Comment:
I'm not sure about this one. It seems logical to allow sorting boolean values, so if you have for example github issues with "closed" boolean flag you can sort the issues to show open issues first and then closed.

Reply:
Yes, that's right. Fixed.